src/server/pkg/testpachd/real_env.go

```diff
@@ -78,6 +78,9 @@ func WithRealEnv(cb func(*RealEnv) error, customConfig ...*serviceenv.PachdFullC
 
 		realEnv.LocalStorageDirectory = path.Join(realEnv.Directory, "localStorage")
 		config.StorageRoot = realEnv.LocalStorageDirectory
+		if err := os.Setenv("STORAGE_BACKEND", "LOCAL"); err != nil {
+			panic(err)
```

Comment:
It's better to return the error so the test process can do cleanup (i.e. in defer blocks), but communicating via env vars is pretty bad because it means we can't do tests in parallel if the environment variable ever has to change.